spleeter_service/spleeter_service.py

```
@@ -0,0 +1,76 @@
+            # Save the uploaded file
+            with open(filename, 'wb') as output_file:
+                output_file.write(file_item.file.read())
+
+            output_dir = 'output'
+            os.makedirs(output_dir, exist_ok=True)
+
+            response = {}
+            try:
+                # Use Spleeter
+                spleeter_command = ['spleeter', 'separate', '-i', filename, '-o', output_dir]
+                subprocess.run(spleeter_command, check=True)
```

Ruff (0.8.0)

`subprocess` call: check for execution of untrusted input

(S603)